src/parser/ir/ir_parser_test.cc

```diff
@@ -30,10 +30,33 @@ TEST_P(IrParserTest, SimpleTestOperation) {
             input_program_text);
 }
 
-INSTANTIATE_TEST_SUITE_P(
-    IrParserTest, IrParserTest,
-    ::testing::Values(
-        "%0 = core.plus []()\n",
-        "%0 = core.minus [access: private, send: yes](<<ANY>>, <<ANY>>)\n"));
+INSTANTIATE_TEST_SUITE_P(IrParserTest, IrParserTest,
+                         ::testing::Values(
+R"(module m0 {
+  block b0 {
+    %0 = core.plus []()
+  }  // block b0
+  block b1 {
+  }  // block b1
+}  // module m0
+)",
+R"(module m0 {
+  block b0 {
+    %0 = core.plus []()
+  }  // block b0
+  block b1 {
+    %1 = core.plus [access: private, transform: no](<<ANY>>, <<ANY>>)
+  }  // block b1
+}  // module m0
+)",
+R"(module m0 {
+  block b0 {
+    %0 = core.plus []()
+  }  // block b0
+  block b1 {
+    %1 = core.plus [access: private, transform: no](%0.out, <<ANY>>)
```

Comment:
Just a clarification. Once you fix ssa names, `%0.out` will simply be `%0`, right?

Reply:
As per my understanding, fixing ssa_names is not enough. `%0.out` here is WA. https://github.com/google-research/raksha/blob/63c0edce07cd17a23f39881911a5e2279ab96397/src/ir/value.h#L53. This part of ToString needs to be changed to print only `%0`.

Reply:
Got it.